src/resources/postcss/tickets-report.pcss

```diff
@@ -20,6 +20,10 @@
 		color: inherit;
 	}
 
+	.welcome-panel-content {
+		min-height: unset;
```

Comment:
Lookgs good but I'd go with `initial`, though.

Reply:
Yep. That works. Thanks!